### ISSUE TITLE
Validate 3.5.0-snapshot Giving Unexpected Errors for Matching Filenames in Separate Directories

### ISSUE THREAD

**Issue by gbowen99** [NASA-PDS-Incubator/validate#873]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I ran validate 3.5.0-snapshot on an InSight documentation bundle the generated report contained a few errors that were not present in the report I had already generated using validate 3.1.1. These additional errors were for file names that were referenced in multiple labels. The two labels are referencing two different sets of files that use the same naming convention. These two sets of images are placed in directories separate from each other, so the according to the latest version of the PDS4 Standards Reference document, I don't think any standards are actually being broken. 

Validation report: 
[insight_documents_report.txt](https://github.com/NASA-PDS/validate/files/14998043/insight_documents_report.txt)

### 🕵️ Expected behavior

I expected no error to be thrown

### 📜 To Reproduce

We do not know exactly how it is run, but here is the report
```
Parameters:
   Targets                      [file:/Z:/data/insight/urn-nasa-pds-insight_documents/]
   Rule Type                    pds4.bundle
   Severity Level               WARNING
   Recurse Directories          true
   File Filters Used            [*.xml, *.XML]
   Data Content Validation      on
   Product Level Validation     on
   Allow Unlabeled Files        false
   Max Errors                   100000
   Registered Contexts File     \\isilon-pri-data\pds-san\stage\tools\pds4_validate\validate-3.5.0-SNAPSHOT\bin\..\resources\registered_context_products.json
```

Note the rule type = `pds4.bundle`

### 🖥 Environment Info

- Validate 3.5.0-snapshot
- Windows Server 2012
- Java OpenJDK version 15.0.2

### 🦄 Related requirements

- 🦄 #755

**Comment by gbowen99** [comment]
Here is the standards doc I referenced: https://pds.nasa.gov/datastandards/documents/sr/v1/StdRef_1.21.0.pdf
Section 6C.1.1 seems to allow what the errors are describing in the validation report.

**Comment by gbowen99** [comment]
Also, the errors I am talking about are for the 9th product. The other errors are unrelated to this issue.

**Comment by al-niessner** [comment]
@jordanpadams @gbowen99

Would it be possible to get a small test data set for this? At a minimum the product that failed (InSight_EastWestMGA.xml) along with its referenced files would work. Run validate on just that product with its isolate dependencies to verify you have the same errors then attach it to this ticket. Thanks.

**Comment by gbowen99** [comment]
Here is the attached report for the test case requested: 
[insight_documents_report_test.txt](https://github.com/NASA-PDS/validate/files/15134317/insight_documents_report_test.txt)

A small clarification is that I included both InSight_EastWestMGA.xml and RISE_InSight_SIS_Raw.xml since these are the two labels that are separately referencing images with the same names in the InSight_EastWestMGA_files and RISE_InSight_SIS_Raw_files image directories.

So the test validation was performed on a sample folder containing only the labels and the two image directories. 

Let me know if you need anything else.

**Comment by jordanpadams** [comment]
@gbowen99 can you point us to this data set online or provide a ZIP of the test data set you created?

**Comment by gbowen99** [comment]
[test_insights_docs_3.5.0.zip](https://github.com/NASA-PDS/validate/files/15151685/test_insights_docs_3.5.0.zip)
Here is the test dataset I used for the report I submitted above.

**Comment by al-niessner** [comment]
Closes does not work from a branch off a branch.
